### DISCUSSION TITLE
Duplicate contigs

### DISCUSSION THREAD

**Discussion by MonicaSteffi** [ablab/spades#1080] [Q&A] [answered]
Dear Developer,

I am using spades to assemble my viral genome with the default setting. In total, I got 5 contigs, two of which are exactly the same. I did compare using blast pairwise alignment and I got 100% identity with 100% query coverage. And also, Blastn results gave the same hit. 

Why do spades generate two contigs exactly the same sequences? How do I interpret this?

Thank you in advance

**Comment by asl** [comment]
Tagging @Dmitry-Antipov

**Comment by Dmitry-Antipov** [comment, marked as answer]
Hi
If you were using --metaviral option, likely it is a known minor bug. See this thread for more detailed discussion and suggestions. https://github.com/ablab/spades/issues/1079
